react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -30,6 +30,16 @@ export default class AbstractPageReloadOverlay extends Component<*, *> {
      * @static
      */
     static propTypes = {
+        /**
+         * The details. The details is an object containing more inforamation
+         * about the connection failed(shard changes, was the computer
+         * suspended, etc.).
```

Comment:
1. Nit: Probably don't need the first sentence that says "The details".
2. Should be changed: inforamation -> information.
3. It doesn't matter: I often don't see type information and public/private for proptype comments.

Only 2 is important to me.